packages/rpc-api/src/simulateTransaction.ts

```diff
@@ -95,7 +107,22 @@ type SimulateTransactionApiResponseWithAccounts<T extends AccountInfoBase> = Sol
     accounts: (T | null)[];
 }>;
 
+type SimulateTransactionApiResponseWithInnerInstructions = SolanaRpcResponse<
+    TransactionForFullMetaInnerInstructionsParsed | TransactionForFullMetaInnerInstructionsUnparsed
+>;
+
 export interface SimulateTransactionApi extends RpcApiMethods {
```

Comment:
I had to double the number of overloads here, to be able to express ‘inner instructions on’ and ‘inner instructions off’ :(

This still keeps us under the (current) max of 24, but as soon as we want to add `replaceRecentBlockhash` to this (next pr) we're hosed.